src/meta/src/manager/system_param/model.rs

```diff
@@ -26,6 +26,9 @@ const SYSTEM_PARAMS_CF_NAME: &str = "cf/system_params";
 pub trait SystemParamsModel: Sized {
     fn cf_name() -> String;
     async fn get<S: MetaStore>(store: &S) -> MetadataModelResult<Option<Self>>;
```

Comment:
Can we forward the implementation of `get` to `get_at_snapshot(store.snapshot())`?